server/seeder/placements.js

```diff
@@ -0,0 +1,68 @@
+const data = {
+  "onboarding": [
+    {
+      "id": Math.random().toString(36).substring(5),
+      "fellow": {
+        "id": "-KXGy1MT1oimjQgFimAv",
+        "email": "korir.kipngetich@andela.com",
+        "first_name": "Ufedojo",
+        "last_name": "Opaluwa",
+        "location": "Lagos",
+        "level": "D3 Developer",
+        "name": "Ufedojo Opaluwa",
+        "picture": "https://lh4.googleusercontent.com/-QOfoONNAWMg/AAAAAAAAAAI/AAAAAAAAADk/Y0ev1giyusI/photo.jpg?sz=50"
+      },
+      "fellow_id": "-KXGy1MT1oimjQgFimAv",
+      "status": "Placed - Awaiting Onboarding",
+      "type": "recommendation",
+      "client_name": "payoff",
+      "placement_mode": "recommendation",
+      "engagement_id": "0061a00000Iggk7AAB",
+      "start_date": "",
+      "end_date": "2017-09-08T13:59:22.868Z",
+      "created_at": new Date(),
+      "updated_at": "2017-09-08T13:59:23.981Z",
+      "updated_by": "Feyikemi Olabiyi",
+      "salesforce_id": "0011a00000CPKyOAAX",
+      "engagement_role_id": "",
+      "engagement_role": null,
+      "next_available_date": "",
+      "sync_status": "",
+      "client_id": "-KXGyJcC1oimjQgFj184"
+    },
+  ],
+  "offboarding": [
```

Comment:
Good jobn. Since `offboarding` data is 99% similar to onboarding except for the `status` field. You can make use of one data and dynamically change the status

Reply:
Will look into this.